modules/on_gke/main.tf

```diff
@@ -248,6 +248,11 @@ resource "helm_release" "forseti-security" {
     value = var.cloud_profiler_enabled
   }
 
+  set {
+    name  = "orchestrator.config.bucket"
```

Comment:
I don't see this variable in the Helm chart's [variables.tf](https://github.com/forseti-security/helm-charts/blob/master/charts/forseti-security/values.yaml). Is there another place that this would be?

Reply:
Not yet implemented.  Working on a PR that would include this.

FWIW, passing a variable to Helm without a receiving variable in the chart is a no-op.  That said, my desire is to have this in master to support the development of the Helm chart PR.

Reply:
BTW, checking for a corresponding value in the values.yaml was the right thing to do.  Thanks for doing that.